pkg/subctl/cmd/join.go

```diff
@@ -130,6 +132,10 @@ func addJoinFlags(cmd *cobra.Command) {
 		"maximum number of packets lost before the connection is marked as down")
 	cmd.Flags().BoolVar(&globalnetEnabled, "globalnet", true,
 		"enable/disable Globalnet for this cluster")
+	cmd.Flags().StringVar(&corednsCustomConfigMap, "coredns-custom-configmap", "",
+		"Name of the custom CoreDNS configmap to configure forwarding to lighthouse")
+	cmd.Flags().StringVar(&corednsCustomNamespace, "coredns-custom-namespace", "",
+		"Name of the namespace in which coredns is running")
```

Comment:
I’m wondering if it would make sense to combine these, *e.g.* `namespace/configmap` (I don’t know if there’s a K8s convention for this)...

Reply:
That would make `namespace` a necessary parameter. Or are you suggesting something like `myconfigmap` or `myns\myconfigmap`

Reply:
I’m suggesting the latter, `namespace/` would be optional. It avoids having to deal with two separate parameters...